src/dlgConnectionProfiles.h

```diff
@@ -66,9 +62,7 @@ public slots:
     void slot_cancel();
     void slot_copy_profile();
 
-
 private:
-    bool removeDir( QString dirName, QString originalPath );
```

Comment:
IIRC This was a compatibility workaround for Qt4.x and early Qt5.x versions - there is/was a duplicate elsewhere (I think in the `Host`class; which couldn't be used in this class because there may not be a `Host` instance to run it from...! :smile:)